hvplot/__init__.py

```diff
@@ -209,6 +209,54 @@ def _hook_patch_docstrings(backend):
 
 @_wraps(_pn.bind)
 def bind(function, *args, **kwargs):
+    """
+    Returns a *reactive* function that *depends* on widgets, parameters or Python objects.
+    This means that the "reactive" function can be automatically invoked whenever the underlying
+    widget or parameter values change. The widgets can be Panel or IpyWidgets.
+    
+    `hv.bind` can be **used to start your `.interactive` pipeline** by loading data depending
+    on widgets or pameters values.
+    
+    Reference: https://github.com/holoviz/hvplot/issues/825
+
+    :Example:
```

Comment:
I've seen you sometimes use `:Example"` and sometimes `Example:`. What's the difference?

Reply:
Is that related in any way to the Numpy docstring convention?

Reply:
I have not given it a lot of thought previously. I've seen different ways of doing it in different places.

I would like to update to 

```rest
Example
---------
```

Because that is in line with the Parameters and Returns section and looks really well organized as a tooltip in VS Code.